### ISSUE TITLE
[a11y-defect-4]: Inconsistently styled headings

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#64358]
### Point of contact

Jamie Klenetsky Fay @coforma-jamie

### Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### Details

`<h3>` headings are styled inconsistently across the form. This can be confusing for sighted users, who use heading styles to better perceive hierarchy on the page.

On [step 1](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/veteran-contact-information), the "Mailing Address" heading uses a font utility class to style it as an `<h4>`:
`<h3 class="vads-u-font-size--h4">Mailing Address</h3>`

On step 3, however, the "How would you like us to contact you?" heading doesn't use any font utility classes:
`<h3>How would you like us to contact you?</h3>`

### Reproduction steps

1. [Go to the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Click "Apply for Veteran Readiness and Employment with VA Form 28-19003" and sign in
3. Navigate to Step 1 of the form, and inspect the "Mailing Address" heading
4. Navigate to Step 3 of the form, and inspect the "How would you like us to contact you?" heading

### Proposed solution or next steps

Pick one way to style `<h3>` across the form, and make sure it's consistent. 

Either:
- Use a font utility class, like `<h3 class="vads-u-font-size--h4">Mailing Address</h3>`, **OR**
- Don't use a font utility class, like `<h3>How would you like us to contact you?</h3>`

### References, articles, or WCAG support

[Subheadings and accessibility](https://www.boia.org/blog/subheadings-and-accessibility-what-is-a-visual-heading)

> "To explain why subheadings must be both visual and programmatically determinable, let’s consider a quick example. 
> 
> A website uses subheading tags (such as `<h2>` and `<h3>`) to structure its content. When a screen reader user accesses the website, they can “scan" the text using the subheading tags to find the information they want. 
> 
> However, the text within the subheading tags looks the same as the other text on the page. People who do not use screen readers or other assistive technologies (AT) cannot quickly identify the headings — in order to find information, they’ll need to read through blocks of text. "

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [X] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [X] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/68020
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/69646
```
